tests/functions/default_arguments.phpt

```diff
@@ -36,9 +36,9 @@ f1(10, 20, 30, 40, 50, 60);
 f1(1, 2, 3, 4, 5, 6, 7);
 //*/
 ///*
-// 2 default followed by one non-default; unusual, but permitted
+// 1 non-default followed by two default; unusual, but permitted
 
-function f2($p1 = 100, $p2 = 1.23, $p3)
+function f2($p1, $p2 = 1.23, $p3 = "abc")
```

Comment:
From the comment, this was supposed to show off that you can have required args after optional args. In that sense, I don't think this signature should change.